src/sso/dsync/CreateDirectory/index.lite.tsx

```diff
@@ -0,0 +1,206 @@
+                  id='google_domain'
+                  name='google_domain'
+                  class={state.classes.input}
+                  onChange={(event) => state.handleChange(event)}
+                  value={state.directory.google_domain}
+                  pattern={`^[a-zA-Z0-9-]+(\.[a-zA-Z0-9-]+)*\.[a-zA-Z]{2,}$`}
+                  title='Please enter a valid domain (e.g: boxyhq.com)'
+                />
+              </div>
+            </Show>
+            <Show when={!props.setupLinkToken}>
+              <>
+                <div class={state.classes.fieldContainer}>
+                  <label for='tenant' class={state.classes.label}>
+                    <span class={defaultClasses.labelText}>Tenant</span>
+                  </label>
+                  <input
+                    type='text'
+                    id='tenant'
+                    name='tenant'
+                    class={state.classes.input}
+                    required={true}
+                    onChange={(event) => state.handleChange(event)}
+                    value={state.directory.tenant}
+                  />
+                </div>
+                <div class={state.classes.fieldContainer}>
+                  <label for='product' class={state.classes.label}>
+                    <span class={defaultClasses.labelText}>Product</span>
+                  </label>
+                  <input
+                    type='text'
+                    id='product'
+                    name='product'
+                    class={state.classes.input}
+                    required={true}
+                    onChange={(event) => state.handleChange(event)}
+                    value={state.directory.product}
+                  />
+                </div>
+              </>
+            </Show>
+            <div class={state.classes.fieldContainer}>
+              <label for='webhook_url' class={state.classes.label}>
+                <span class={defaultClasses.labelText}>Webhook URL</span>
+              </label>
+              <input
+                type='text'
+                id='webhook_url'
+                name='webhook_url'
+                class={state.classes.input}
+                onChange={(event) => state.handleChange(event)}
+                value={state.directory.webhook_url}
+              />
+            </div>
+            <div class={state.classes.fieldContainer}>
+              <label for='webhook_secret' class={state.classes.label}>
+                <span class={defaultClasses.labelText}>Webhook secret</span>
+              </label>
+              <input
+                type='text'
+                id='webhook_secret'
+                name='webhook_secret'
+                class={state.classes.input}
+                onChange={(event) => state.handleChange(event)}
+                value={state.directory.webhook_secret}
+              />
+            </div>
+            <div>
+              <button class={state.classes.button} type='submit'>
```

Comment:
@Ryukemeister Please use the `Button` component (it has the BoxyHQ theme) here and in other places.